lib/formtools/plugins/plugins-helpers.js

```diff
@@ -252,6 +242,50 @@ var filtersDefault = function (filters) {
 
 };
 
+var exportDefault = function (exports) {
+
+    if (exports === undefined || exports === null) {
+        return undefined;
```

Comment:
Yes, I amended my commit and force pushed. I was naughty. I just wanted to give it a try. Whoops.